core/src/main/java/uk/ac/bbsrc/tgac/miso/core/store/QcTargetStore.java

```diff
@@ -14,4 +15,6 @@
   public Collection<? extends QC> listForEntity(long id) throws IOException;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1452)